lib/theme/toast_messages/toast_messages_extension.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/theme/toast_messages/hs_toast_theme.dart';
+
+extension ToastMessagesExtension on BuildContext {
+  void showToast(ToastType type, String title, String message) {
+    ScaffoldMessenger.of(this).showSnackBar(SnackBar(
+      content: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Padding(
+              padding: const EdgeInsets.only(top: 12),
+              child: ToastMessageContainer(
+                  toastType: type, toastTitle: title, toastContent: message))
+        ],
+      ),
+      backgroundColor: Colors.transparent,
+      behavior: SnackBarBehavior.floating,
+      elevation: 0,
+      duration: const Duration(seconds: 6),
+    ));
+  }
+
+  void closeToast() {
```

Comment:
Why do u need to expose closeToast out of Toast? Toast will close only if user taps on x button within toast